docs/reference/snapshot-restore/apis/restore-snapshot-api.asciidoc

```diff
@@ -183,7 +183,8 @@ state then the restore request will fail.
 If `include_global_state` is `true`, the request restores all feature states
 in the snapshot by default. If `include_global_state` is `false`, the request
 restores no feature states by default. To restore no feature states, regardless
```

Comment:
Again, I'd explicitly state that an empty array results in default behavior here.